src/gwt/acesupport/acemode/markdown_highlight_rules.js

```diff
@@ -64,11 +64,10 @@ var ScalaHighlightRules = require("ace/mode/scala_highlight_rules").ScalaHighlig
 var ShHighlightRules = require("mode/sh_highlight_rules").ShHighlightRules;
 var StanHighlightRules = require("mode/stan_highlight_rules").StanHighlightRules;
 var SqlHighlightRules = require("mode/sql_highlight_rules").SqlHighlightRules;
-var JavaScriptHighlightRules = require("ace/mode/javascript_highlight_rules").JavaScriptHighlightRules;
```

Comment:
Why was this removed? (unused var?)

Reply:
It was accidentally duplicated - it's already defined a few lines up.